zerver/lib/webhooks/common.py

```diff
@@ -114,3 +133,25 @@ def validate_extract_webhook_http_header(request: HttpRequest, header: str,
         raise MissingHTTPEventHeader(header)
 
     return extracted_header
+
+
+def get_http_headers(integration_name: str, fixture_name: str) -> Dict["str", "str"]:
+    """ This method is just a helper that finds and executes the correct fixture_to_headers method
+    for the given integration which would in turn generate the actual headers themselves. """
```

Comment:
I rewrote this comment to focus on the background for what this does for someone who hasn't heard of this function.  

As a sidenote, "This method is just a helper that" is filler text that shouldn't appear in comments or documentation; in general, we want to just say what one needs to know in a focused, clear fashion.

```
    """For integrations that require custom HTTP headers for some (or all)                       
    of their test fixtures, this method will call a specially named                              
    function from the target integration module to determine what set                            
    of HTTP headers goes with the given test fixture.                                            
```

Reply:
Got it, I'll keep this in mind in future PRs. Thanks for the tip!